module9/ImageSearch/ImageSearch/Views/SearchResultsView.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  SearchGrid.swift
+//  ImageSearch
+//
+//  Created by Peter Rice on 2/28/24.
+//
+
+import SwiftUI
+
+struct SearchResultsView: View {
+  @StateObject var viewModel = ImageSearchViewModel(imageStore: ImageSearchStore())
+
+  var body: some View {
+    NavigationStack {
+      ScrollView {
+        VStack {
+          ResultsGridView(viewModel: viewModel)
+          if viewModel.showMoreImagesButton {
+            Button("More Images") {
+              viewModel.moreImagesTapped()
+            }
+          }
+          if viewModel.searchLoadingState == .loadingSearch {
+            ProgressView()
+          }
+        }
+      }
+      .padding()
+      .navigationTitle(Text("Plexel Images"))
+      .navigationDestination(for: PlexelImage.self) { imageResult in
+        ImageDetailsView(viewModel: viewModel, imageResult: imageResult)
+      }
+      .searchable(
+        text: $viewModel.searchQuery,
+        placement: .navigationBarDrawer(displayMode: .always),
+        prompt: "Search")
+      .onSubmit(of: .search) {
+        viewModel.searchInputSubmitted()
+      }
+    }
+  }
+}
+
+struct ResultsGridView: View {
+  @ObservedObject var viewModel: ImageSearchViewModel
+
+  @Environment(\.isSearching) var isSearching
```

Comment:
Getting a SwiftLint error here:

Attributes Violation: Attributes with arguments or inside always_on_line_above must be on a new line instead of the same line (attributes)

Not sure what this means.